### DISCUSSION TITLE
Idea de diagrama de flujo para practica de toma de requerimientos.

### DISCUSSION THREAD

**Discussion by dmejia96** [Launch-X-Latam/MisionFrontEnd#130] [Show and tell]
¡Hola! Les presento la idea del flujo de proceso para el punto 3 de la práctica de requerimientos.
¡Retroalimentaciones son bien recibidas!
Espero ver sus ideas.
https://www.figma.com/file/loDBkzCnIWYGHJQdyzmpLS/ABOGABOT-FLOW-CHART?node-id=0%3A1

**Comment by Wyspect** [comment]
¡Hola! Buenas madrugadas.

Me gusta la propuesta, no tengo mucha experiencia pero no sería mejor:

1. Inicio
2. Crear cuenta/verificación
3. formulario
4. pago

Una vez confirmado el pago el formulario debería enviarse para el documento de word automatizado, o bueno, asi lo estructuraria yo.

-Saludos,

**Reply by dmejia96** [reply]
Si, tenía un poco de conflicto con eso ya que al momento de llenar formulario si te falta o no puedes hacer un paso de verificación tal vez pierdas la información del formulario y por ende tiempo.
Muchas gracias por tu comentario voy a cambiar el orden.
Saludos! Que tengas excelente día.

**Reply by AndrjsM** [reply]
Hola @dmejia96 opino igual que @Wyspect el primer paso debería ser el registro/inicio de sesión, luego la toma de datos y por último el proceso de pago.

**Reply by dmejia96** [reply]
@Wyspect muchas gracias por la idea de la validación cada cierto tiempo, sin duda es una idea genial!

**Comment by LuceroLuciano** [comment]
Hola.

Gracias por compartir, te quedo increible! la idea de negocio se entiende claramente, 10/10. 
Solo tengo una duda, en la parte del formulario se va directo al apartado de pago, y después se crea la cuenta,  estaba pensando y según yo primero seria la creación de la cuenta  que nos llevaría al llenado del formulario para después poder realizar un pago en caso de que el cliente desee hacer una demanda., algo así como la pagina de Amazon, donde primero nos registramos llenando un formulario con todos nuestros datos, y ahí es donde se crea nuestra cuenta porque pide un correo y una contraseña, una vez dentro de la pagina de amazón podemos ver los productos y realizar una compra, que nos lleva a un método de pago pero es cuando ya tenemos la cuenta, es decir no podemos hacer un pago sino tenemos una cuenta. Espero me haya dado a entender, es una idea vaga que me surgió. 

Espero que me ayuden a entender mejor esta parte porque tengo esa inquietud. 


Saludos.

**Reply by dmejia96** [reply]
Hola! Claro, es una buena idea, de hecho lo voy a modificar en este momento y ya para que quede asentada esa propuesta que me gustó mucho. Saludos!

**Comment by JuanSposada** [comment]
Muy bien. Me gustó. Muy sencillo y digerible. Me gusta. Exito

**Reply by dmejia96** [reply]
Gracias compañero! Espero estés disfrutando el curso.

**Comment by dmejia96** [comment]
Hola compañeros! Ya lo actualicé, tal vez de esta manera se entienda mejor y el proceso sea más lógico, gracias por su ayuda!
Si en algo puede o puedo ayudarles con gusto.
